app/views/announcements/show.html.erb

```diff
@@ -26,22 +26,26 @@
   </p>
 
   <div class="flex gap-2">
-    <% if ((current_user == @announcement.author && organizer_signed_in?(as: :manager)) || auditor_signed_in?) && !editing %>
-      <% if @announcement.draft? %>
+    <% if !editing %>
+      <% if @announcement.draft? && (policy(@announcement).publish? || auditor_signed_in?) %>
```

Comment:
hmm - why are we showing this to auditors but not giving them permission?

Reply:
My thought process was that in most places on HCB, auditors can see all actions available but the buttons are disabled for them if they do not have permission, so I did the same for these buttons.